templates/provider/kcm/templates/rbac/user-facing/clusteripam-editor.yaml

```diff
@@ -0,0 +1,15 @@
+# permissions for end users to edit managementbackups.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    k0rdent.mirantis.com/aggregate-to-global-admin: "true"
+  name: {{ include "kcm.fullname" . }}-ipam-editor-role
+rules:
+- apiGroups:
+  - k0rdent.mirantis.com
+  resources:
+  - clusteripamclaims
+  - clusteripamclaims/status
```

Comment:
The admin (or whoever with the appropriate permission) has access only to the claims in terms of RW access, that is the expected behavior, right?

Reply:
That's right. I'm assuming the admin would have the viewer role too right?